Controller/ResourceController.php

```diff
@@ -39,11 +46,99 @@ public function __construct(
         $this->registry = $registry;
     }
 
-    public function resourceAction($repositoryName, $path)
+    public function getResourceAction($repositoryName, $path)
     {
         $repository = $this->registry->get($repositoryName);
         $resource = $repository->get('/'.$path);
 
+        return $this->createResponse($resource);
+    }
+
+    /**
+     * There are two different changes that can currently be done on a cmf resource:
+     *
+     * - move
+     * - rename
+     *
+     * changing payload properties isn't supported yet.
+     *
+     * @param string $repositoryName
+     * @param string $path
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function patchResourceAction($repositoryName, $path, Request $request)
+    {
+        $repository = $this->registry->get($repositoryName);
+        $this->failOnNotEditable($repository, $repositoryName);
+
+        $path = '/'.ltrim($path, '/');
+
+        $requestContent = json_decode($request->getContent(), true);
+        if (!$requestContent) {
+            return $this->badRequestResponse('Only JSON request bodies are supported.');
+        }
+
+        foreach ($requestContent as $action) {
+            switch ($action['operation']) {
```

Comment:
do we really need that operation? Is there no chance to change the path of the resource information or the target path of the request to use that as an information? to have some kind of operation key in the body is not very RESTful.